openid-sharedsignals-framework-1_0.md

```diff
@@ -694,6 +697,27 @@ The following are hypothetical examples of SETs that conform to the Shared Signa
 ~~~
 {: #subject-custom-type-ex title="Example: SET Containing an SSF Event with a Proprietary Subject Identifier Format"}
 
+~~~ json
+{
+  "iss": "https://idp.example.com/",
+  "jti": "756E69717565206964656E746966696572",
+  "iat": 1520364019,
+  "txn": 8675309,
+  "aud": "636C69656E745F6964",
+  "sub_id": {
+    "format": "phone_number",
+    "phone_number": "+1 813 867 5309"
+  },
+  "events": {
+    "https://schemas.openid.net/secevent/caep/event-type/session-revoked": {}
+  },
+  "actions": {
```

Comment:
If a profile is just changing processing semantics of the secevent but not changing the event itself, why is there a need to extend the event with an `actions` claim?

Reply:
I strongly disagree that we should be extending the secevent schema to support mandatory actions uris.  I thought the goal of #255 was just to allow profiles to define additional processing semantics of an event not change the event itself to describe actions.  I made similar comments on the referenced issued.

Reply:
I agree with Karl here. The `actions` field was a proposed normative way that we could make SSF prescriptive. I think the intent of this PR, at this stage of v1, is just to add some text that says "If you want to use Shared Signals in a prescriptive way, the SSF police won't come after you about it." I think your paragraph above does that well and this example just confuses things.